src/fa/game_session.py

```diff
@@ -177,13 +187,45 @@ def _turn_state_changed(self, val):
 
     def _launched(self):
         self._logger.info("Game has started")
+        self._add_to_full_gamelog()
+
+    def _limit_replayid_gamelogs(self):
+        nb_replayid_files=0
+        list_game_replayid_files=os.listdir(util.LOG_DIR)
+        list_game_replayid_files.sort()
+        for element in list_game_replayid_files[::1]:
```

Comment:
How about `replay_files = [e for e in list_game_replayid_files if "Replayid" in e]`?

Reply:
Sure, fixed